zanata-war/src/main/resources/messages.properties

```diff
@@ -1168,6 +1168,8 @@ jsf.email.admin.UserMessageIntro=Zanata user "{0}" with id "{1}" has sent the fo
 jsf.email.ReplyInstructions=You can reply to {0} at {1}
 jsf.email.admin.ReceivedReason=You are an administrator in the system configuration
 jsf.email.admin.user.ReceivedReason=You are an administrator
+jsf.email.admin.ReceivedReason.language.noCoordinator=There is no coordinator for language "{0} ({1})"
+jsf.email.admin.ReceivedReason.versionGroup.noMaintainer=There is no maintainer for version group "{0}"
```

Comment:
Suggest:

 - This message is intended for the language coordinators of {0} ({1}). No coordinators are assigned to {0} so this will be sent to server administrators instead.
- This message is intended for the maintainers of version group "{0}". No maintainers are assigned to version group "{0}" so this will be sent to server administrators instead.